packages/node_modules/@ciscospark/plugin-lyra/src/occupants.js

```diff
@@ -0,0 +1,178 @@
+/**!
+ *
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {SparkPlugin} from '@ciscospark/spark-core';
+import querystring from 'querystring';
+import {base64} from '@ciscospark/common';
+
+/**
+ * @class
+ * @extends {Lyra}
+ * @memberof Lyra
+ */
+const Occupants = SparkPlugin.extend({
+  namespace: `Lyra`,
+
+  /**
+   * Joins a lyra space, update every 10 minutes to keep alive for MANUAL
+   * @param {Types~LyraSpace} space
+   * @param {string} space.url
+   * @param {object} options
+   * @param {string} options.passType
+   * @returns {Promise}
+   */
+  join(space, options = {passType: `MANUAL`}) {
```

Comment:
`spark.lyra.spaces.join(space)` would be consistent with the conversation plugin. I would expect the first parameter of methods of the occupants plugin to be an `occupant`.